src/Library/demos/Welcome/main.js

```diff
@@ -1,6 +1,7 @@
 import Gtk from "gi://Gtk?version=4.0";
 
 Gtk.init();
+Adw.init();
```

Comment:
`Adw.init()` already calls `Gtk.init()` so it should be one or the other.

This particular demo doesn't use `Adw`

Reply:
That is right. But the style is different then.

Reply:
Should we always use `Adw.init()` ?